src/Hbar.ts

```diff
@@ -37,29 +36,37 @@ export const hbarUnitSymbols = {
     gigabar: 'Gℏ',
 };
 
+/**
+ * Typesafe wrapper for values of HBAR providing foolproof conversions to other denominations.
+ */
 export class Hbar {
-    /** The Hbar value in tinybar, used natively by the SDK and Hedera itself */
+    /** The HBAR value in tinybar, used natively by the SDK and Hedera itself */
     private readonly tinybar: BigNumber;
 
     private constructor(tinybar: BigNumber) {
-        checkNumber(tinybar);
         this.tinybar = tinybar;
     }
 
     /**
-     * Calculate the Hbar amount given a raw value and a unit
-     * (assumes value is in hbar if not provided)
+     * Calculate the HBAR amount given a raw value and a unit.
      */
-    public static from(amount: number | BigNumber | string, unit: HbarUnit = 'hbar'): Hbar {
+    public static from(amount: number | BigNumber | string, unit: HbarUnit): Hbar {
```

Comment:
We agreed earlier to drop the default here, it just didn't make it into the original PR.

Reply:
Since this wasn't published yet this shouldn't be a breaking change.